subprojects/snapshots/src/main/java/org/gradle/internal/vfs/impl/DefaultVirtualFileSystem.java

```diff
@@ -178,6 +187,7 @@ private CompleteFileSystemLocationSnapshot readLocation(String location) {
 
     @Override
     public void update(Iterable<String> locations, Runnable action) {
+        recentlyCreatedSnapshotsListener.snapshotsCreated(locations);
```

Comment:
Are they created or updated?

Reply:
Currently, they are invalidated. At some point, we wanted to store the recently created things in there: https://github.com/gradle/gradle/issues/12761

Let's do renaming together.